HISTORY.md

```diff
@@ -1,4 +1,81 @@
-# 0.40.5
+# 0.42.0
+
+## Breaking Changes
+
+**AdvancedVI 0.5**
+
+Turing.jl v0.42 updates `AdvancedVI.jl` compatibility to 0.5.
+Most of the changes introduced in `AdvancedVI.jl@0.5` are structural, with some changes spilling out into the interface.
+The summary of the changes below are the things that affect the end-users of Turing.
+For a more comprehensive list of changes, please refer to the [changelogs](https://github.com/TuringLang/AdvancedVI.jl/blob/main/HISTORY.md) in `AdvancedVI`.
+
+  - A new level of interface for defining different variational algorithms have been introduced in `AdvancedVI` v0.5. As a result, the method `Turing.vi` now receives a keyword argument `algorithm`. The object `algorithm <: AdvancedVI.AbstractVariationalAlgorithm` should now contain all the algorithm-specific configurations. Therefore, keyword arguments of `vi` that were algorithm-specific such as `objective`, `operator`, `averager` and so on, have been moved as fields of the relevant `<: AdvancedVI.AbstractVariationalAlgorithm` structs.
```

Comment:
Sorry to be picky, but could I ask for a code example of a typical use of `vi`? Something like

> If you previously did
> ```
> vi(mymodel, blahblah; objective=o, operator=or, average=ar)
> ```
> that would now be done instead with
> ```
> vi(mymodel, blahblah; algorithm=blah(o, or, ar)
> ```

I think your explanation already says this, but just trying to really hold the user by the hand through the breakage.

Reply:
This is a bit tricky because the interface now does not fully expose all of the controls that were previously available. Let me think if there's something I can do.